packages/next/next-server/server/image-optimizer.ts

```diff
@@ -44,30 +43,31 @@ export async function imageOptimizer(
     return { finished: true }
   }
 
-  let absoluteUrl: URL
-  try {
-    absoluteUrl = new URL(url)
-  } catch (_error) {
-    // url was not absolute so assuming relative url
+  let absoluteUrl: URL | undefined
+  let relativeUrl: string | undefined
+
+  if (url.startsWith('/')) {
```

Comment:
We should also support relative urls that don't start with `/` (although I think I forgot to add a test for that).

Reply:
As discussed in chat, it seems like this would be better to handle in the image component and have the API always expect the leading slash for relative URLs since that no leading slash makes it relative to the current page which the API isn't aware of without relying on the `referrer` header